x-pack/plugins/security_solution/public/common/containers/matrix_histogram/index.ts

```diff
@@ -99,6 +118,7 @@ export const useMatrixHistogram = ({
                     refetch: refetch.current,
                     totalCount: response.totalCount,
                   }));
+                  setMatrixHistogramBuckets(histogramBuckets);
```

Comment:
is it possible to add the `histogramBuckets` inside of the MatricHistogramResponse to avoid a new re-render with a new state.

Reply:
Yes! Moved it per your suggestion here: https://github.com/elastic/kibana/pull/78985/commits/aa4af5f193de460e8f2885a31444204d427ef579